ci/templates/.travis.yml

```diff
@@ -1,61 +0,0 @@
-dist: xenial
-language: python
-cache: false
-env:
-  global:
-    - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
-    - SEGFAULT_SIGNALS=all
```

Comment:
is this libSegFault/SEGFAULT_SIGNALS=all something important that needs to be ported to GHA?

Reply:
ah you asked this in the PR description:

> * I left out these global env vars, are they needed?
> 
> ```yml
> env:
>   global:
>     - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
>     - SEGFAULT_SIGNALS=all
> ```

Reply:
seems like it was introduced here https://github.com/pytest-dev/pytest-cov/pull/58/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R6